IMDA-BTG/aiverify
# bugfix - set width to widget description text to prevent overlap, and…

## Description

Minor quick fixes - 
- overlapping plugin description text in plugins screen
- fix regex in plugins search reported in https://github.com/IMDA-BTG/aiverify/security/code-scanning/23 


## Type of Change



- fix: A bug fix









## How to Test

1. plugin description should not overlap (see screenshot in [issue 136](https://github.com/IMDA-BTG/aiverify/issues/136)
2. test plugin search. It should work as before.

![image](https://github.com/IMDA-BTG/aiverify/assets/130328114/83cd3627-fc25-4997-b9f3-dd79e8862885)


## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x ]  I have tested the changes locally and verified that they work as expected.
- [x ]  I have rebased my branch onto the latest commit of the main branch.
- [x ]  I have squashed or reorganized my commits into logical units.
- [x ]  I have performed a self-review of my own code.
- [ x]  I have read, understood and agree to the Developer Certificate of Origin below, which this project utilises.

## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]

<details>
  <summary>Developer Certificate of Origin</summary>

 ```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
 ```
</details>
